src/qt/textctrl.cpp

```diff
@@ -61,6 +62,16 @@ void wxQtLineEdit::returnPressed()
     }
 }
 
+void wxQtLineEdit::keyPressEvent(QKeyEvent *event)
+{
+    wxTextCtrl *handler = GetHandler();
+    if ( !event->text().isEmpty() && maxLength() == handler->GetValue().length() )
```

Comment:
Could you please add a comment explaining why is the first part of the test needed?

Also, this:
```suggestion
    if ( !event->text().isEmpty() && maxLength() == handler->GetLastPosition() )
```
would be more efficient, which might matter here.

Reply:
@vadz,

GetLastPosition() is not yet implemented for wxTextCtrl under Qt, and so for now I think I will use the current code